### ISSUE TITLE
[Bug]: RAM usage warning via email

### ISSUE THREAD

**Issue by RubenKelevra** [netdata/netdata#13043]
### Bug description

I get a lot of ram warnings via email. The module analyzing the memory usage isn't taking the used memory of the ARC by ZFS into account - which is substantial on this machine.

Additionally, if there would be really a low memory issue on the machines the PSI (Pressure Stall Information) for the memory would be non-zero. But it is zero nearly all the time.

So it would be nice if the memory analyzing plugin could honor the ARC cache in memory and do count it like regular cache.

Additionally I would recommend only sending messages out if there's really a memory pressure event (if PSI is available) and not if there's just low memory in general, with no large demand for more.

### Expected behavior

Analyze the situation more thoroughly with taking ARC and Memory Pressure into account.

### Steps to reproduce

1. Use ZFS
2. Use Netdata
3. Get messages

### Installation method

native binary packages (.deb/.rpm)

### System info

```shell
$ # Linux
uname -a; grep -HvE "^#|URL" /etc/*release
Linux odin.pacman.store 5.17.5-arch1-1 #1 SMP PREEMPT Wed, 27 Apr 2022 20:56:11 +0000 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID="Arch"
/etc/lsb-release:DISTRIB_RELEASE="rolling"
/etc/lsb-release:DISTRIB_DESCRIPTION="Arch Linux"
/etc/os-release:NAME="Arch Linux"
/etc/os-release:PRETTY_NAME="Arch Linux"
/etc/os-release:ID=arch
/etc/os-release:BUILD_ID=rolling
/etc/os-release:ANSI_COLOR="38;2;23;147;209"
/etc/os-release:LOGO=archlinux-logo
```

### Netdata build info

```shell
$ netdata -W buildinfo
Version: netdata v1.34.1
Configure options:  '--prefix=/usr' '--sbindir=/usr/bin' '--sysconfdir=/etc' '--libexecdir=/usr/lib' '--localstatedir=/var' '--with-zlib' '--with-math' '--with-user=netdata' 'CFLAGS=-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -flto=auto' 'LDFLAGS=-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now -flto=auto' 'CXXFLAGS=-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -flto=auto'
Install type: custom
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  YES
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    NO
    IPMI:                    NO
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 YES
    Prometheus Remote Write: YES
```

**Comment by ilyam8** [comment]
Hi, @RubenKelevra. This is a duplicate of #12843, fixed in #12847. Will be in the next stable release.

**Comment by RubenKelevra** [comment]
Thanks for the fast response! Yeah didn't found that via the search 🤔

Thanks for fixing
